data/json/recipes/recipe_ammo.json

```diff
@@ -117,11 +117,11 @@
   "reversible": false,
   "autolearn": true,
   "book_learn": [[ "glassblowing_book", 5 ]],
+  "qualities":[
+    {"id":"GRIP","level":3}
```

Comment:
I think the suggestion from @illi-kun was to avoid manual reformatting (#18523) and I asked to avoid mixing formatting and semantic changes in the same commit (#18536).